qt/aqt/profiles.py

```diff
@@ -309,6 +309,9 @@ def backupFolder(self) -> str:
     def collectionPath(self) -> str:
         return os.path.join(self.profileFolder(), "collection.anki2")
 
+    def log_folder(self) -> str:
+        return self._ensureExists(os.path.join(self.base, "logs"))
```

Comment:
Is this only used by add-ons? If so, maybe something like addon_logs would make more sense?

Reply:
Should we move `anki.log` into the folder perhaps? Happy to do so in a follow-up, or update the method name otherwise.

Reply:
Most users will have an empty anki.log, due to the default log level. I don't think I've taken advantage of it once since it was first introduced, and would be just as happy to see it go, as there are very few issues that can't be better diagnosed with a traceback or a copy of the collection.

@cav71 would you mind renaming the method + folder name? Then I'll get this merged in.